### ISSUE TITLE
Using "each" and "merge" cause problem

### ISSUE THREAD

**Issue by Younsion** [jxlsteam/jxls#62]
hi，jxls team
when i am using 'each' and 'mergeCells' at the same time，i found the problem below here。
my template：
![image](https://user-images.githubusercontent.com/26084893/96421858-e3d0c080-1229-11eb-9c2e-63f2eb877f6e.png)

what i want：
![image](https://user-images.githubusercontent.com/26084893/96422067-22667b00-122a-11eb-83f2-f85562401168.png)

But what I actually got is：
![image](https://user-images.githubusercontent.com/26084893/96422246-5cd01800-122a-11eb-98b1-8f9c555b37cd.png)

am i wrong or it is a bug？
waiting for your reply...thank you.

**Comment by SoltauFintel** [comment]
I don't know. But I see that you have 5 extra rows and have a +5 in your formula!?

**Comment by Younsion** [comment]
yes...as you see,what i want is the piture 2,the each-loop include the '分组平均水平'(the extra 5 rows),so i do '+5' in the merge command...

**Comment by SoltauFintel** [comment]
Because of Corona, we are currently spending significantly less time on JXLS. Therefore an answer or processing can take longer.

**Comment by SoltauFintel** [comment]
You could upload an example template and a piece of source code. But please with no Chinese chars.

**Comment by Younsion** [comment]
hi,here is a example of it:
[sample.zip](https://github.com/jxlsteam/jxls/files/5462897/sample.zip)

**Comment by howiewant** [comment]
这个问题解决了吗

**Comment by gastendonk** [comment]
Nein.

**Comment by fitzyyf** [comment]
I just happened to encounter the same problem, which might be an issue. It could be that after merging the cells , the ﻿mergeCells command adjusts the Size, not using the default template cell Size. This leads to an increase in row or column index labels during subsequent loops, which results in new rows. You can customize a merge command that only performs the merge without changing the columns.
```java
 @Override
  public Size applyAt(CellRef cellRef, Context context) {
    int rows = getVal(this.rows, context);
    int cols = getVal(this.cols, context);

    if (rows > 1 || cols > 1) {
      getTransformer().mergeCells(cellRef, rows, cols);
    }
    area.applyAt(cellRef, context);
    return area.applyAt(cellRef, context);
  }
```
